frontend/src/App.tsx

```diff
@@ -1,10 +1,11 @@
+import React from 'react'
 import { useRoutes } from 'react-router-dom'
 import { ThemeProvider } from 'styled-components'
 import { Dashboard } from './pages/dashboard/index'
 import { theme } from './theme/index'
-import { Landing } from './pages/landing/index'
-import { Login } from './pages/login/index'
-import { Register } from './pages/register/index'
+import  Landing  from './pages/landing/index'
+import  Login  from './pages/login/index'
+import  Register  from './pages/register/index'
```

Comment:
If you're using "index" files for exports, you can omit the "index" part on imports. So all these imports could be:
`import  Landing  from './pages/landing'`